src/main.cc

```diff
@@ -93,6 +94,9 @@ int main(int argc, char* argv[]) {
   // Load singletons and global variable.
   config = new configuration::state;
 
+  // Hack to instanciate the logger.
+  configuration::applier::logging::instance();
+
```

Comment:
do you want to keep this ?

Reply:
Yes. Otherwise you won't have any error log return.